packages/modules/desktop-viewer-react/src/services/Initializer.ts

```diff
@@ -45,9 +47,15 @@ export class DesktopInitializer {
       this._initializing = true;
 
       const cancellable = makeCancellable(function* () {
+        ViewerPerformance.enable(options?.enablePerformanceMonitors);
+        ViewerPerformance.addMark("ViewerStarting");
         const additionalRpcInterfaces = options?.additionalRpcInterfaces ?? [];
         additionalRpcInterfaces.push(SnapshotIModelRpcInterface);
 
+        if (!options) {
```

Comment:
is this chunk neded?

Reply:
it is as the code stands since we spread the options below. I went back and forth between making options not optional and leaving optional and assuming false here. My rationale is that it is still a required prop of the component that calls this function, but we don't have to change the function and caller this way. Definitely open to discussion here. This is the first required initialization parameter that we've added.

Reply:
i _think_ we can make options required in the desktop viewer too now, which could help us eliminate this chunk

Reply:
fixed